high_performance_computing/hpc_parallel_intro/01_introduction.md

```diff
@@ -0,0 +1,399 @@
+---
+name: Introduction to Parallelism
+dependsOn: []
```

Comment:
Sorry for the delayed response, ill *again*. Episode 10 on scalability does because it requires you to profile on a HPC cluster, but the rest works as an entirely local course. We could add a callout at the top saying that section needed `hpc_intro` as a prereq? 

I think there's a lot of use cases for wanting to learn how to write parallel code *before* you learn how to run on a cluster, so I think leaving it untagged lets the material support the two most common pathways into HPC: 
* Parallelising your code locally then moving it onto a cluster.
* Running other people's parallel code on a cluster then learning how to modify it.

Reply:
Added a callout to the top of the MPI optimisation episode.